pkg/operator/ceph/pool/controller.go

```diff
@@ -341,11 +347,31 @@ func (r *ReconcileCephBlockPool) reconcileCreatePool(clusterInfo *cephclient.Clu
 	return reconcile.Result{}, nil
 }
 
+func getCephName(pool cephv1.CephBlockPool) (string, error) {
+	if pool.Name == "" {
+		return "", errors.Errorf("metadata name must be set")
```

Comment:
```suggestion
		return "", errors.New("metadata name must be set")
```

Reply:
Isn't that checked by the API server already?

Reply:
yes, I didn't have it at first, and added it for unit test completeness. But I think i'll remove it again since it's overkill.